website/src/models/group.js

```diff
@@ -404,11 +425,12 @@ GroupSchema.methods.leave = function(user, keep, mainCb){
           group.type === 'party' ||
           (group.type === 'guild' && group.privacy === 'private')
       )){
-        // TODO remove invitations to this group
         groupWasRemoved = true;
-        Group.remove({
+        Group.findOne({
```

Comment:
I don't think we need to `find` the document again, we can just to `group.remove()` as it's already an instance of Group